server/app/api/v1/line.py

```
@@ -31,103 +32,130 @@
 logger = logging.getLogger(__name__)
 settings = get_settings()
 UPLOAD_DIR = settings.line_audio_files_dir
 MAX_FILES = settings.line_max_audio_files
+user_image_cache: Dict[str, bytes] = {}
 
 @router.post("/callback/{agent_id}")
 async def handle_line_callback(
     request: Request,
-    agent_id: str, 
+    agent_id: str,
     llm_service: LLMService = Depends(get_llm_service),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)